pySDC/implementations/problem_classes/AllenCahn_1D_FD.py

```diff
@@ -224,21 +230,24 @@ def eval_f(self, u, t):
         self.work_counters['rhs']()
         return f
 
-    def u_exact(self, t):
+    def u_exact(self, t, u_init=None, t_init=None):
```

Comment:
Sorry for nitpicking, but putting `u_init` and `t_init` in the signature implies that you can use this function to compute a local error, when really you can't. The function will ignore these variables and give the analytical exact solution starting from t=0 regardless. So I suggest replacing them with `**kwargs` to avoid confusion. Also there is a typo in the doctoring regarding `u_init` below.

Reply:
Don't really like that the signature of `u_exact` can be different depending on the problem, it breaks genericity. 
Also, I'm not sure that the signature should indicate what the method does, when comparing two methods that have the same name.

Actually, I thought that those `u_init` and `t_init` argument could be used to reduce computation time, when computing reference solution with scipy on many time points, which have no impact on `u_exact` method using an analytical reference solution but then is kept for consistency.

I would prefer indicating in the docstring that those `u_init` and `t_init` are not used. Modifying this with **kwargs add more ambiguity than everything else. But if you really disagree with keeping those, then I'll simply remove it and adapt the project code using the `u_exact` generically with a `try` and `catch` block ...

Reply:
Did not receive any comment from your side, so I'll just comply and remove the `u_init` and `t_init` from the signature (I spend quite some time on this PR, this should be merged at some point to submit the paper).

But this just delay some serious discussion we should have at some point about uniformizing the problem classes ...